docs/current_docs/cookbook/cookbook.mdx

```diff
@@ -175,21 +293,75 @@ When working with private Git repositories, ensure that [SSH authentication is p
 
 - Clone the `main` branch of the public `dagger/dagger` GitHub repository to `/src` in the container:
 
+    <Tabs groupId="shell">
+    <TabItem value="System shell">
+    ```shell
+    dagger -c 'clone https://github.com/dagger/dagger BRANCH main'
+    ```
+    </TabItem>
+    <TabItem value="Dagger Shell">
+    ```shell title="Type 'dagger' to start Dagger Shell in interactive mode and enter the command below."
+    clone https://github.com/dagger/dagger BRANCH main
+    ```
+    </TabItem>
+    <TabItem value="Dagger CLI">
     ```shell
     dagger call clone --repository=https://github.com/dagger/dagger --locator=BRANCH --ref=main
     ```
+    </TabItem>
+    </Tabs>
 
 - Clone the public `dagger/dagger` GitHub repository at tag `v0.12.1` to `/src` in the container and list the contents of the `CHANGELOG.md` file in the repository root directory:
 
+    <Tabs groupId="shell">
+    <TabItem value="System shell">
     ```shell
-    dagger call clone --repository=https://github.com/dagger/dagger --locator=TAG --ref=v0.12.1 file --path=/src/CHANGELOG.md contents
+    dagger <<EOF
+    clone https://github.com/dagger/dagger TAG v0.12.1 |
+      file /src/CHANGELOG.md |
+      contents
+    EOF
     ```
+    </TabItem>
+    <TabItem value="Dagger Shell">
+    ```shell title="Type 'dagger' to start Dagger Shell in interactive mode and enter the command below."
+    clone https://github.com/dagger/dagger TAG v0.12.1 | file /src/CHANGELOG.md | contents
```

Comment:
General check for "Dagger Shell" tabs having the same multiple breaks as "System shell" EOF.